src/components/Display.js

```diff
@@ -0,0 +1,9 @@
+import React, { useState } from "react";
+
+const Display = props => (
```

Comment:
I think you also need some css in this PR because otherwise this component won't look as the display in the calculator.